data-otservbr-global/lib/others/vip_system.lua

```diff
@@ -31,7 +31,7 @@ function Player.onRemoveVip(self)
 		self:setOutfit(playerOutfit)
 	end
 
-	self:setStorageValue(Storage.VipSystem.IsVip, 0)
+	self:kv():remove("vip")
```

Comment:
acredito que seja melhor usar um scoped neste caso, já que é um sistema e vip é um nome muito genérico, pode conflitar com outra key existente, sugiro isto:
```suggestion
	self:kv():scoped("account"):remove("vip-system")
```